server/settings_test.go

```diff
@@ -655,6 +706,63 @@ func TestSettings(t *testing.T) {
 				})
 			})
 
+			t.Run("STTCheckIntervalsOK", func(t *testing.T) {
```

Comment:
`STTCheckIntervalsOK` -> `STTCheckIntervalsValid` since the previous test is named `..Invalid`

Reply:
Done